### DISCUSSION TITLE
Integrate brc-20 native management

### DISCUSSION THREAD

**Discussion by GaloisField2718** [ordinals/ord#2086] [Ideas]
Hello everyone, 

I created this post to chat with you about a possible integration of brc-20 management natively by `ord` command. 

It could looks something like this `ord wallet brc20 list` to get all brc-20 in wallet. 
Moreover, we can think about preset to `deploy`, `mint`, and `transfer` brc-20.

Especially, `transfer` is quite tricky actually because if I want to transfer a brc-20 token `tick` I need to inscribe a `tick` transfer and send this inscription. Maybe it can be a good tool to manage all of this natively in command line interface. 

I don't know what do you think about this feature and I'm wondering what your point of view is about this ?

I'm totally available to chat deeper with you and try to develop this soon. 

See you soon 💥

**Comment by tyjvazum** [comment]
@GaloisField2718, I'm willing to add support to [arb](https://github.com/tyjvazum/arb) if you'd like to open an issue to discuss a possible design.

**Reply by GaloisField2718** [reply]
Hello, I really didn’t know this ! A pleasure.

**Comment by stet** [comment]
I suggest using arb as well for this. Afaik, Casey/Ord are not interested in BRC-20, understandably. But maybe sentiment has changed. 

Still, it makes much more sense to not use ord protocol specifier for BRC-20. I have proposed namespaced protocol specifier some months ago and would like to explore this over on arb.

**Reply by GaloisField2718** [reply]
Hi, I’m totally agree with you about the possible disinterest of the ord team about this feature and it should be easier to develop this on ˋarb`. 
but once it’s deployed on arb it should obviously be more integrated if the feature is also on ˋord`. 
Because I think many people will continue to use ord instead of arb. 
But you’re right, let’s begin on ˋarb`. Have you already develop something on top of this project ?

**Reply by stet** [reply]
> Hi, I’m totally agree with you about the possible disinterest of the ord team about this feature and it should be easier to develop this on ˋarb`. but once it’s deployed on arb it should obviously be more integrated if the feature is also on ˋord`.
> Because I think many people will continue to use ord instead of arb.
> But you’re right, let’s begin on ˋarb`. Have you already develop something on top of this project ?

Just getting started this (June).

**Comment by artrepreneur** [comment]
Can any of the existing BRC20 wallets be utilized as a standalone, non-web based wallet. I would prefer to not even use a api, as in unisat, but just a RPC. Any suggestions?

**Reply by tyjvazum** [reply]
I'm not aware of anything that's available already, but I'm working on providing that in [arb](https://github.com/tyjvazum/arb/issues/14).

**Comment by VivekKadvani** [comment]
Hello, I'm seeking a solution for BRC20. I'm looking for code that can assist me in deploying, minting, and sending BRC20 tokens. Is there a solution available for this?

**Reply by GaloisField2718** [reply]
Hello, 

As far as I know no integrated solution exists to manage brc20 in cli. As it shouldn't be too hard to implement, you can try first an easy python code for example, to write JSON in good format and use subprocess.run() to call ˋord wallet inscribe {file}ˋ.

**Comment by VivekKadvani** [comment]
Hello, I'm seeking a solution for BRC20. I'm looking for code that can assist me in deploying, minting, and sending BRC20 tokens. Is there a solution available for this?
